index.js

```diff
@@ -0,0 +1,36 @@
+const express = require("express");
+const videoRouter = require("./routes/video-router.js");
```

Comment:
You can import the module without giving extension .js

i.e const videoRouter = require("./routes/video-router");

it will also work